formats/application/vnd/oada/isoblue/1/+json/schema.js

```diff
@@ -0,0 +1,21 @@
+const libvocab = require('vocabs/oada');
```

Comment:
Unsure if this is the best way to organize the ISOBlue schema. To my understanding, the alternative is to have a `formats/isoblue/1/+json/...` folder for this and a `formats/isoblue-data/${data}` for the location/can/heartbeat data. Hoping to keep the number of folders to a minimum as well as avoid fragmenting the isoblue stuff too much

Reply:
Per in person review with Ault, folder structure is okay